src/sagemaker/model.py

```diff
@@ -40,6 +40,9 @@
         "rk3288",
         "rk3399",
         "sbe_c",
+        "aisage",
```

Comment:
what is this? why it is related to BAH?

Reply:
It's neo instance type updates. Shall we remove it from this PR?